### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ddohler**
Updates the README with expanded instructions based on working through deployment of a new stack yesterday. Most of this information was already available somewhere, so this pulls together existing sources, expands certain points, and reorganizes the README to be in roughly the order required when setting up a new stack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,80 @@
...
+You will be asked to provide a password and some information about yourself and your organization.
+
+Next, you will also need to generate a certificate from the keystore. To do this, run:
+`keytool -exportcert -keystore driver.keystore -alias driver -file driver_android_certificate.pem`
+
+The keystore file should be kept private since it is used to verify the authenticity of the Android
+app that you build. The certificate file should be placed on a publicly accessible web server.
+GitHub Pages is a good option.
+
+You should copy the keystore file to the `gradle/data` file of your DRIVER installation and
```
Comment (flibbertigibbet) [inline README.md]: Might consider linking to the relevant section of the DRIVER README here.

**Review by flibbertigibbet — APPROVED** [review]
:+1: 

Great notes. Would be worth noting that the added material is generally only relevant for deploying; for development purposes, copying over the two template files and setting the server URL should be sufficient.

**Review by kshepard — APPROVED** [review]
Looks great.

**Comment by ddohler** [comment]
Updated to address comments.

**Pull request merged**